cookie-banner-rules-list.json

```diff
@@ -5996,19 +5934,6 @@
       "id": "b62e5259-f7f5-4319-8175-e39266b8a031",
       "domains": ["healthline.com"]
     },
-    {
-      "click": {
-        "optIn": "button#onetrust-accept-btn-handler",
-        "optOut": "button#onetrust-reject-all-handler",
-        "presence": "div#onetrust-consent-sdk"
-      },
-      "cookies": {
-        "optIn": [{ "name": "_tt_enable_cookie", "value": "1" }],
```

Comment:
Same thing here, can't this just be merged with the main onetrust rule?

Reply:
Tested the cookies and merged the rules accordingly.